src/vmspawn/vmspawn.c

```diff
@@ -211,7 +211,7 @@ static int parse_argv(int argc, char *argv[]) {
 static int run_virtual_machine(void) {
         _cleanup_(ovmf_config_freep) OvmfConfig *ovmf_config = NULL;
         _cleanup_strv_free_ char **cmdline = NULL;
-        _cleanup_free_ char *machine = NULL, *qemu_binary = NULL, *mem = NULL;
+        _cleanup_free_ char *machine = NULL, *qemu_binary = NULL, *mem = NULL, *kcl = NULL;
```

Comment:
Please reduce the scope of this variable, i.e. move it into the if block.

Reply:
Fix included in: 20602d9a36.